src/TxHistory/ModalInfo/ModalInfo.tsx

```diff
@@ -39,10 +38,3 @@ const useStrings = () => {
     infoTitle: intl.formatMessage(messages.infoTitle),
   }
 }
-
-const styles = StyleSheet.create({
-  infoIcon: {
-    alignSelf: 'center',
```

Comment:
I think this will change the alignment from `center` to `left`.

Reply:
yeah you are totally right